core/src/replay_stage.rs

```diff
@@ -665,6 +674,10 @@ impl ReplayStage {
             let blockhash = bank.last_blockhash();
             vote_tx.partial_sign(&[node_keypair.as_ref()], blockhash);
             vote_tx.partial_sign(&[voting_keypair.as_ref()], blockhash);
+            // before publishing the latest vote on the network, send the tower to the tower snapshot service
```

Comment:
Comment needs to be updated. Tower snapshot service isn't a thing right?

Reply:
oops 😊